charter/index.html

```diff
@@ -238,23 +244,20 @@ <h3>
             The Working Group will deliver the following W3C normative specifications:
           </p>
                 <dl>
-                    <dt id="solid-protocol" class="spec">Solid Protocol v1.0</dt>
+                    <dt id="PUMPKIN-protocol" class="spec">PUMPKIN Protocol v1.0</dt>
```

Comment:
This is the place where it might make sense to continue to use "Solid".

Reply:
Or, even better - just name it "a protocol specification", name Solid as input below, and then you can easily absorb it and call your protocol spec Solid without raising hackles over pre-disposing toward it now.

Reply:
@cwilso, exactly. I think that's currently covered in this PR in that the group proposes to work on a protocol where Solid Protocol (ED) will serve as one possible "input document" (see paragraph below in the charter). Background: https://github.com/solid/solid-wg-charter/pull/63#issuecomment-1931796745 based on past work/experience.

"PUMPKIN Protocol" or "a protocol specification" as per your suggest both work IMO.

Aside: Now, the reality is that while the Solid Protocol may be the leading input at this point in time, the charter is mindful of https://www.w3.org/Guide/standards-track/#criteria and so other works qualifying as such would be welcome as input towards "a protocol specification". For instance, while the Fedora API Specification 1.0 ( https://fcrepo.github.io/fcrepo-specification/ ) wasn't incubated in a W3C CG, it emerged from an open process with a reasonable dose of "strong and broad consensus" and implementations that are in use today. IMO, such works (that are also compatible with the Solid Protocol, e.g., LDP, and in fact using Web Access Control: https://solid.github.io/web-access-control-spec/ ) would qualify as input, but that's for the potential WG to decide.

Reply:
I am at a bit of a loss to understand what the Fedora spec *IS*, exactly, in standards terms.  It claims to be a Recommendation/PR/CR, but it is not in any W3C group that I can tell? 

My net point here was that the WG might have a deliverable that ends up being called Solid, particularly presuming the WG chooses to adopt the current Solid protocol and use it.  Your Working Group should NOT be called Solid, because it doesn't mean anything, and you don't have that community together already to make such a decision.

Reply:
@cwilso the Fedora Specification is a self-published document that comes from an organization called [Lyrasis](https://www.lyrasis.org/) and not part of any official standards body (e.g. W3C, IETF). I was involved in that work ~7 years ago. The hope at the time was that it would result in multiple, interoperable implementations of the Fedora API specification, but that never really materialized. AFAIK, there is only a [single implementation](https://fedora.lyrasis.org/), but I have also not been involved in quite some time.

The Fedora API is an interesting case in that it extends the LDP specification by adding features that are important to a particular domain (digital repositories). Those features include access control, notifications and resource versioning, which happen to overlap with the current Solid protocol. I think the point @csarven is making here is that, like the current Solid API protocol, the Fedora API involved an open process outside of an official standards body.

To @cwilso's net point: the Fedora project is also an interesting case because the API specification _and_ the reference implementation had the same name: Fedora. That led to various types of confusion, and I'm not sure it helped the project overall.

Reply:
* The Fedora API and the Solid Protocol share similarities in that they both can serve as inputs toward the protocol specification outlined in the charter. It is understood that the list of input documents is uncountable and additional works could be acknowledged to support the goal.
* Needless to say, the Solid Protocol has been incubated within the W3C CG since 2018. Although the Fedora API was not incubated within W3C space, it originated from an open-source community and process. (Their use of the Rec/CR/PR terminology has no particular weight / meaning at W3C.)

@cwilso, at least the Solid CG (based on discussions on record) shares your view that the WG will be called whatever it needs to be. We've moved on from proposing the "Solid" WG, hence this PR. Furthermore, given that the Solid Protocol is just one input document, I wouldn't rule out the possibility that the protocol specification produced by the WG may call it something entirely different. That's the intention behind "PUMPKIN Protocol v1.0".

So, perhaps we can just clarify the placeholder names:

the "Smashing WG" and the "Pumpkins Protocol"

:)

Reply:
@csarven Love it!  "W3C and the Infinite Sadness"

Reply:
@acoburn Thanks for the elucidation.  I think I was confused because Fedora is presented at places as if it's a W3C REC-track document (PR, etc), but afaict, it's not ever been a work item in W3C (even in a CG Note, I think, but I could be wrong there).

Reply:
> @cwilso, at least the Solid CG (based on discussions on record) shares your view that the WG will be called whatever it needs to be. We've moved on from proposing the "Solid" WG, hence this PR. Furthermore, given that the Solid Protocol is just one input document, I wouldn't rule out the possibility that the protocol specification produced by the WG may call it something entirely different. That's the intention behind "PUMPKIN Protocol v1.0".

Pointer to this would be good.  Still trying to get my head round this.  So PUMPKIN is an umbrella to Solid and specs like solid?  Makes not a huge amount of sense to me, as it would be taking on more, and I think Solid would work best as a REC if it took on a core subset, rather than, pivot to a superset.  Some context would be helpful.

Reply:
Resolving this; as csarven pointed out, it may be that the final name of the protocol is neither Solid nor the name of the working group.